girder_worker_utils/tee.py

```diff
@@ -0,0 +1,161 @@
+import sys
+
+
+def tee_stdout(cls):
```

Comment:
Let's document this method and `tee_stderr`, namely to say that it is meant to be used to decorate subclasses of `Tee`.